fixit/cli/add_new_rule.py

```diff
@@ -86,32 +87,48 @@ def create_rule_file(file_path: Path, rule_name: str) -> None:
     print(f"Successfully created {file_path.name} rule file at {file_path.parent}")
 
 
-def main() -> None:
-    parser = argparse.ArgumentParser(
-        description="Creates a skeleton of adding new rule file"
-    )
+def _add_arguments(
+    parser: Union[argparse._SubParsersAction, argparse.ArgumentParser]
+) -> None:
```

Comment:
The passed parser is simply an `ArgumentParser`.
```suggestion
def _add_arguments(
    parser: argparse.ArgumentParser
) -> None:
```

Reply:
Done, thank you